cms/envs/test.py

```diff
@@ -129,9 +129,17 @@
     },
 }
 
-# This hack disables migrations during tests. We want to create tables directly from the models for speed.
-# See https://groups.google.com/d/msg/django-developers/PWPj3etj3-U/kCl6pMsQYYoJ.
-MIGRATION_MODULES = {app: "app.migrations_not_used_in_tests" for app in INSTALLED_APPS}
+if os.getenv('DISABLE_MIGRATIONS'):
```

Comment:
@nedbat i have added comments but don't think i addressed what you were asking for.  If the change I just made allows the build to pass I can amend the comment to better explain what the technique is doing.